modules/administration_manual/examples/installation/apps_management_installation/move-non-core-shipped-apps-to-custom-dir.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+APPS_LIST=$(sudo -u www-data ./occ app:list --shipped=false --output=json)
```

Comment:
not all distros use www-data as user, is this script intended for a specific distro ?

Reply:
yes, it was decided to focus in docs on Ubuntu